src/libraries/Common/src/System/Security/Cryptography/MLDsa.cs

```diff
@@ -1140,14 +1073,12 @@ public static MLDsa ImportMLDsaSecretKey(MLDsaAlgorithm algorithm, ReadOnlySpan<
             ThrowIfNotSupported();
             ArgumentNullException.ThrowIfNull(algorithm);
```

Comment:
Same feedback as in SLH-DSA - the `ThrowIfNull` should come before `ThrowIfNotSupported`

And also a reminder that when this gets OOBed, you won't be able to use this throw helper.